setup.py

```diff
@@ -51,6 +51,7 @@
     'sendgrid==5.6.0',
     'simple-crypt==4.1.7',
     'unicodecsv==0.14.1',
+    'xxhash==1.4.1',
```

Comment:
Have you looked into the hash functions in hashlib for python? They hash to the same value by default. Installing a new library just for hashing might not be ideal.

```
import hashlib
hashlib.algorithms_available
{'sha3_256', 'shake256', 'sha3_512', 'ripemd160', 'sha512', 'sha384', 'md5', 'shake_256', 'md5-sha1', 'md4', 'whirlpool', 'sha3-512', 'blake2b512', 'sha224', 'shake_128', 'sha1', 'sha3-384', 'blake2b', 'shake128', 'sha3-256', 'sha512-256', 'sha3-224', 'sha256', 'sm3', 'blake2s256', 'sha3_384', 'blake2s', 'sha512-224', 'sha3_224'}
```

Reply:
Updated to use blake2b instead.